zerver/views/registration.py

```diff
@@ -363,28 +377,76 @@ def accounts_register(request: HttpRequest) -> HttpResponse:
     return render(
         request,
         'zerver/register.html',
-        context={'form': form,
-                 'email': email,
-                 'key': key,
-                 'full_name': request.session.get('authenticated_full_name', None),
-                 'lock_name': name_validated and name_changes_disabled(realm),
-                 # password_auth_enabled is normally set via our context processor,
-                 # but for the registration form, there is no logged in user yet, so
-                 # we have to set it here.
-                 'creating_new_team': realm_creation,
-                 'password_required': password_auth_enabled(realm) and password_required,
-                 'require_ldap_password': require_ldap_password,
-                 'password_auth_enabled': password_auth_enabled(realm),
-                 'root_domain_available': is_root_domain_available(),
-                 'default_stream_groups': get_default_stream_groups(realm),
-                 'accounts': get_accounts_for_email(email),
-                 'MAX_REALM_NAME_LENGTH': str(Realm.MAX_REALM_NAME_LENGTH),
-                 'MAX_NAME_LENGTH': str(UserProfile.MAX_NAME_LENGTH),
-                 'MAX_PASSWORD_LENGTH': str(form.MAX_PASSWORD_LENGTH),
-                 'MAX_REALM_SUBDOMAIN_LENGTH': str(Realm.MAX_REALM_SUBDOMAIN_LENGTH)
-                 }
+        context = {
+            'form': form,
+            'email': email,
+            'key': key,
+            'full_name': request.session.get('authenticated_full_name', None),
+            'lock_name': name_validated and name_changes_disabled(realm),
+            # password_auth_enabled is normally set via our context processor,
+            # but for the registration form, there is no logged in user yet, so
+            # we have to set it here.
+            'creating_new_team': realm_creation,
+            'billing_enabled': settings.BILLING_ENABLED,
+            'password_required': password_auth_enabled(realm) and password_required,
+            'require_ldap_password': require_ldap_password,
+            'password_auth_enabled': password_auth_enabled(realm),
+            'root_domain_available': is_root_domain_available(),
+            'default_stream_groups': get_default_stream_groups(realm),
+            'accounts': get_accounts_for_email(email),
+            'MAX_REALM_NAME_LENGTH': str(Realm.MAX_REALM_NAME_LENGTH),
+            'MAX_NAME_LENGTH': str(UserProfile.MAX_NAME_LENGTH),
+            'MAX_PASSWORD_LENGTH': str(form.MAX_PASSWORD_LENGTH),
+            'MAX_REALM_SUBDOMAIN_LENGTH': str(Realm.MAX_REALM_SUBDOMAIN_LENGTH)
+        }
     )
 
+def notify_support_new_realm(user_profile: UserProfile, organization_type: str,
```

Comment:
Why are we passing in `user_profile`, rather than `realm`? It looks like the function only cares about the `realm`.